docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -5,57 +5,96 @@ permalink: /documentation/using-testcafe/common-concepts/browser-support.html
+For more information and examples, see:
 
-## Specifying Browsers for Test Task
+* Command line: [Local Browsers](../command-line-interface.md#local-browsers)
+* API: [runner.browsers](../programming-interface/runner.md#browsers)
 
-If you are working with TestCafe through the command line,
-[list the desired browsers](../command-line-interface.md#browser-list)
-within the `testcafe` command.
+## Portable Browsers
 
-In case you are using the API, specify the browsers
-with the [runner.browsers](../programming-interface/runner.md#browsers) method.
+To use a portable browser, specify the path to the browser executable file. For more information and examples, see:
 
-To identify browsers, different approaches are used.
+* Command line: [Portable Browsers](../command-line-interface.md#portable-browsers)
+* API: [runner.browsers](../programming-interface/runner.md#browsers)
 
-Browser&#160;Type  | Specified by
------------------- | ----------------------------------
-Local browser      | The path to the browser executable or browser short name. For the full list of names, see the table in the [Directly Supported Browsers](#directly-supported-browsers) section. To obtain the list of browsers installed on the current machine, run TestCafe from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
-Browser&#160;accessed&#160;through a&#160;[browser&#160;provider&#160;plugin](../common-concepts/browser-support.md#browser-provider-plugins). | The browser alias that consists of the browser provider name and the name of a browser itself.
-Remote browser     | The `remote` alias or [BrowserConnection](../programming-interface/browserconnection.md).
\ No newline at end of file
+## Browsers on Remote Devices
+
+To run tests on a remote mobile and desktop device, this device must have network access to the TestCafe server.
+
+First, you will need to create a remote browser connection.
+
+* Command line: specify the `remote` *alias* (see [Remote Browsers](../command-line-interface.html#remote-browsers))
+* API: use the [createBrowserConnection](../programming-interface/testcafe.html#createbrowserconnection) method
+
+After that, TestCafe will provide a URL to open on the remote device in the browser against which you want to test. As you open this URL, the browser connects to the TestCafe server and starts testing.
+
+## Browsers in Cloud Testing Services
+
+TestCafe allows you to use browsers from cloud testing services. You can access them through [browser provider plugins](../../extending-testcafe/browser-provider-plugin/).
+
+The following plugins for cloud services are currently provided by the TestCafe team.
+
+Service                              | Plugin
+------------------------------------ | -------------------
+[Sauce Labs](https://saucelabs.com/) | [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
```

Comment:
That's my version of Browser Support, so I'll explain...

Yes, I do think that we need to clearly list which cloud providers we support. Not just "for example... search further for yourself". Each new provider is a good feature and when we support, say, BrowserStack, I would like to have it listed somewhere in the docs.

BTW, the parallel with Yeoman is not a good one.